src/App.js

```diff
@@ -1,3 +1,49 @@
-export default function App() {
-  return <div>App goes here</div>;
-}
+import {useState} from 'react';
+import styled from 'styled-components';
+import Header from './components/Header';
+import Form from './components/Form';
+import TodosList from './components/TodosList';
+
+const App = () => {
+  const [input, setInput] = useState('');
+  const [todos, setTodos] = useState([]);
+  return (
+    <div className="container">
+      <div className="app-wrapper">
+        <div>
+          <Header />
+        </div>
+        <Styleddiv>
+          <TodosList todos={todos} setTodos={setTodos} />
+        </Styleddiv>
+        <Styleddiv2>
```

Comment:
StyledDiv2 is not a good name. You'd better think about which element to show it. For example, your div shows a card, so the name could be StyledCard.